internal/proxy/README.md

```diff
@@ -17,48 +17,6 @@ dependency to the proxy:
 1.  Ensure that the new dependency is covered under one of the `notice`,
     `permissive`, or `unencumbered` licences under the categories defined by
     [Google Open Source](https://opensource.google.com/docs/thirdparty/licenses/).
-2.  Gather the new set of dependencies and sync them to the GCS bucket.
+2.  Gather the new set of dependencies and sync them to the GCS bucket by running
+    `internal/proxy/add.sh` from the root of the Go Cloud repository.
```

Comment:
I recommend adding a sentence like the following to show that this is an interactive script:

> Review the files to be sent to the proxy when prompted by the script.

Reply:
Done.